FullStats failts on python 3

circus 0.12.1 (from pip)
python 3.4.3

Running at local and at Heroku:

2015-12-18 17:05:25 tornado.application[30] [ERROR] Exception in callback <bound method FullStats.look_after of <circus.plugins.statsd.FullStats object at 0x7f5abcf5cf60>>
 Traceback (most recent call last):
   File "/app/.heroku/python/lib/python3.4/site-packages/tornado/ioloop.py", line 1039, in _run
     return self.callback()
   File "/app/.heroku/python/lib/python3.4/site-packages/circus/plugins/statsd.py", line 128, in look_after
     self.statsd.gauge("_stats.%s.watchers_num" % name, len(cpus))
   File "/app/.heroku/python/lib/python3.4/site-packages/circus/plugins/statsd.py", line 37, in gauge
     self.send(bucket, "%s|g" % value)
   File "/app/.heroku/python/lib/python3.4/site-packages/circus/plugins/statsd.py", line 26, in send
     self.socket.sendto("%s:%s" % (bucket, value), (self.host, self.port))
 TypeError: 'str' does not support the buffer interface

--- a/circus/plugins/statsd.py
+++ b/circus/plugins/statsd.py
@@ -23,7 +23,8 @@ class StatsdClient(object):
         if self.prefix:
             bucket = "%s.%s" % (self.prefix, bucket)
 
-        self.socket.sendto("%s:%s" % (bucket, value), (self.host, self.port))
+        msg = "%s:%s" % (bucket, value)
+        self.socket.sendto(msg.encode('utf-8'), (self.host, self.port))
 
     def decrement(self, bucket, delta=1):
         if delta > 0:

